lib/uplink/uplink.go

```diff
@@ -70,10 +69,6 @@ type Config struct {
 		// DialTimeout is the maximum time to wait connecting to another node.
 		// If not set, the library default (20 seconds) will be used.
 		DialTimeout time.Duration
-
-		// RequestTimeout is the maximum time to wait for a request response from another node.
-		// If not set, the library default (20 seconds) will be used.
-		RequestTimeout time.Duration
```

Comment:
Maybe we should keep this parameter, without logic behind, and mark it as deprecated?

Reply:
:+1: We at least need to be sure that supplying the option via command-line doesn't crash the program if it's removed.

Reply:
this is the point of the volatile section though. i'm also okay with people finding out their config flag is gone this way.